### PR TITLE
homework_6

### PR DESCRIPTION
**Pull request by niampire**

### CODE REVIEW

File: homeworks/kateryna.chepurna_niampire/homework_6/task_6_1.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+function UserInfo() {
+    let fullName;
+    let phone;
+    Object.defineProperties(this, {
+        fullName: {
+            get() {
+                return fullName;
+            },
+            set(name) {
+                fullName = name.trim().toLowerCase().replace(/\s+/g, ' ').replace(/\b\w/g, letter => letter.toUpperCase());
```
Comment (markoffden) [inline homeworks/kateryna.chepurna_niampire/homework_6/task_6_1.js]: `replace(/\s+/g, ' ')` - replace whitespace with whitespace? cool :) but why is it needed here?

File: homeworks/kateryna.chepurna_niampire/homework_6/task_6_1.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+function UserInfo() {
```
Comment (markoffden) [inline homeworks/kateryna.chepurna_niampire/homework_6/task_6_1.js]: camelCase naming here please

Comment (niampire) [reply homeworks/kateryna.chepurna_niampire/homework_6/task_6_1.js]: renamed, but eslint in vs code said me that require constructor names to begin with a capital letter (new-cap)

File: homeworks/kateryna.chepurna_niampire/homework_6/task_6_1.js (hunk 1 above)
Comment (niampire) [reply homeworks/kateryna.chepurna_niampire/homework_6/task_6_1.js]: I delete this regexp, but I wanted to remove more then one whitespace.

**Pull request merged**